File: java/sprint1_nonfinals/C.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.stream.Collectors;
import java.util.List;
import java.util.Comparator;
import java.io.IOException;

public class C {

    private static List<Integer> getNeighbours(List<List<Integer>> matrix, int row, int col) {
        // Ваше решение
    }

    public static void main(String[] args) throws IOException {
        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
            int rowsCount = readInt(reader);
            int colsCount = readInt(reader);
            List<List<Integer>> matrix = readMatrix(reader, rowsCount);
            int rowId = readInt(reader);
            int colId = readInt(reader);
            List<Integer> neighbours = getNeighbours(matrix, rowId, colId);
            for (int element : neighbours) {
                System.out.print(element + " ");
            }
        }
    }


    private static int readInt(BufferedReader reader) throws IOException {
        return Integer.parseInt(reader.readLine());
    }

    private static List<Integer> readList(BufferedReader reader) throws IOException {
        return Arrays.asList(reader.readLine().split(" "))
                .stream()
                .map(elem -> Integer.parseInt(elem))
                .collect(Collectors.toList());
    }

    private static List<List<Integer>> readMatrix(BufferedReader reader, int rowsCount) throws IOException {
        List<List<Integer>> matrix = new ArrayList<>(rowsCount);
        for (int i = 0; i < rowsCount; i++) {
            matrix.add(readList(reader));
        }
        return matrix;
    }
}

File: swift/sprint1_nonfinals/c.swift
func getNeighbours(_ matrix: [[Int]], _ row: Int, _ col: Int) -> [Int] {
    // Здесь реализация вашего решения
}


func readInts() -> [Int] {
    return readLine()!.split(separator: " ").compactMap { Int($0)! }
}

func readInt() -> Int {
    return Int(readLine()!)!
}

func print(_ result: [Int]) {
    print(result.compactMap{ String($0) }.joined(separator: " "))
}

func main() {
    let n = readInt()
    let m = readInt()
    let matrix = (1...n).map{ _ in readInts() }
    let row = readInt()
    let col = readInt()
    print(getNeighbours(matrix, row, col))
}

main()
